pkg/processor/spec_processor.go

```diff
@@ -14,9 +14,9 @@ import (
 )
 
 const (
-	// . is a valid character in a name, cannot use as a separator.
```

Comment:
Well, I thought resource name in a bundle should follow the same naming guidelines but I'm fine with `#`.